css/style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  background: #1c1a19;
+}
+
+.img {
+  position: absolute;
+  width: 406.89px;
+  height: 312.03px;
+  left: 620.1px;
+  top: 317px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.topnav {
```

Comment:
[OPTIONAL]

You could also select the HTML elements from the `<header>` tag, that way you wouldn't need to use a new div and class :thinking: